### ISSUE TITLE
NTR: Glycine cleavage enzyme activity in fibroblasts low (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8232]
New term request Glycine cleavage enzyme activity in fibroblasts low (elthat_160327140358):   UMLS: 4227590

**Comment by pnrobinson** [comment]
@MickeySegal
which of the four enzymes is meant? (see e.g. https://pubmed.ncbi.nlm.nih.gov/18941301/)
Is this a clinical assay?

**Comment by MickeySegal** [comment]
It sounds like the cleavage system is assessed as a whole:

https://www.omim.org/entry/614462
"Patient cells also showed absent activity of the glycine cleavage enzyme system."

https://www.omim.org/entry/616859
"Both patients, as well as the patient reported by Wei et al. (2011), had decreased glycine cleavage enzyme activity."

https://www.omim.org/entry/614299
"Biochemical studies showed impaired enzymatic activity of the glycine cleavage system and deficient pyruvate dehydrogenase (PDH) activity, both consistent with a defect in lipoate biosynthesis."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000829
- label: Reduced tissue glycine cleavage enzyme activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:18941301
- def: Concentration or activity of the glycine cleavage enzyme below the lower limit of normal. This enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts.
- comment: Biochemically, the glycine cleavage enzyme system is composed of the P-protein (GLDC gene, MIM# 238300), which removes CO2 from glycine and transfers the amino-methyl group to lipoate on the H-protein (GCSH gene, MIM# 238330), and the T-protein (AMT gene, MIM# 238310) which releases ammonia and forms methylenetetrahydrofolate, after which the reduced lipoate is re-oxidized by the L-protein.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8232
- synonyms:
